OpenRA.Game/HotkeyDefinition.cs

```diff
@@ -48,8 +48,9 @@ public HotkeyDefinition(string name, MiniYaml node)
 				Types = FieldLoader.GetValue<HashSet<string>>("Types", typesYaml.Value);
 
 			if (nodeDict.TryGetValue("Contexts", out var contextYaml))
-				Contexts = FieldLoader.GetValue<HashSet<string>>("Contexts", contextYaml.Value)
-					.Select(c => ContextFluentPrefix + "." + c).ToHashSet();
+				Contexts = [
```

Comment:
Cases like this, where we go from straightforward left-to-right mental parsing to having to read the inner expression and then mentally note the spread / collection syntax is doing the collection type change.

Reply:
When it's multi-line it can be a bit crazy